rust/kernel/lib.rs

```diff
@@ -21,7 +21,8 @@
     const_panic,
     const_raw_ptr_deref,
     const_unreachable_unchecked,
-    try_reserve
+    try_reserve,
+    bool_to_option
```

Comment:
Do we need `bool_to_option`?

Reply:
Oh, not anymore since we are not using `then_some` anymore. Thanks

Reply:
Fixed by 92a19c2ef2d0d0780c34327b273a21db627af711